priv/repo/migrations/20230619091028_add_tenant_ip_version.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Supavisor.Repo.Migrations.AddTenantIpVersion do
+  use Ecto.Migration
+
+  def up do
+    alter table("tenants", prefix: "_supavisor") do
+      add(:ip_version, :string, null: false)
```

Comment:
Have you consider adding a [constraint](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#constraint/3) to this column? I think this can better guarantee the consistency of data by checking that the value can only be `v4`, `v6` or `auto`. It's a behaviour similar to what would be if you the "enum" database type was used instead of string. WDYT?

Reply:
Also, it would be nice to have a default value in the DB column as well. Maybe "auto"? WDYT?

Reply:
good call, I added these updates in the last commit https://github.com/supabase/supavisor/pull/115/commits/6fdfdbb5afa07612408ea96e86bc3fe30e6f95bc